### ISSUE TITLE
Explain PE more

### ISSUE THREAD

**Issue by ba55ie** [vicompany/front-end#1]
E.G. the difference in PE for a informative website versus a web app.
https://nolanlawson.com/2016/10/13/progressive-enhancement-isnt-dead-but-it-smells-funny/

**Comment by timseverien** [comment]
I'm all for researching PE, especially since our clients have interesting use-cases that often depend on JavaScript support.

> E.G. the difference in PE for a informative website versus a web app.

This example, however, gives me the shivers! Websites and web apps are both very, very generic categories of web-based products. Effectiveness of PE is very dependant on a product's features and purpose. I fear that a recommendation reduces that awareness and limits creativity for searching the optimal solution. `</rant>`

TL;DR: tread with caution.

**Comment by ba55ie** [comment]
There are indeed a lot of grays between the black & white of PE. 😃  The link in the first comment is more of a reference because a lot of pros and cons are being described and discussed in that article and the comments.

**Comment by ba55ie** [comment]
Added a link to https://www.gov.uk/service-manual/technology/using-progressive-enhancement
